internal/sbi/producer/nf_management.go

```diff
@@ -93,16 +93,21 @@ func HandleUpdateNFInstanceRequest(request *httpwrapper.Request) *httpwrapper.Re
 func HandleGetNFInstancesRequest(request *httpwrapper.Request) *httpwrapper.Response {
 	logger.NfmLog.Infoln("Handle GetNFInstancesRequest")
 	nfType := request.Query.Get("nf-type")
-	limit, err := strconv.Atoi(request.Query.Get("limit"))
-	if err != nil {
-		logger.NfmLog.Errorln("Error in string conversion: ", limit)
-		problemDetails := models.ProblemDetails{
-			Title:  "Invalid Parameter",
-			Status: http.StatusBadRequest,
-			Detail: err.Error(),
-		}
+	limit_param := request.Query.Get("limit")
+	limit := 0
+	if limit_param != "" {
+		var err error
+		limit, err = strconv.Atoi(request.Query.Get("limit"))
+		if err != nil {
+			logger.NfmLog.Errorln("Error in string conversion: ", limit)
+			problemDetails := models.ProblemDetails{
+				Title:  "Invalid Parameter",
+				Status: http.StatusBadRequest,
+				Detail: err.Error(),
+			}
 
```

Comment:
I think if a client set limit_param, which is lower than 1, NRF should return the response with a Bad request.

Reply:
Sounds right, I will update the PR tomorrow.